lockbox-ios/Presenter/FxAPresenter.swift

```diff
@@ -5,26 +5,29 @@
 import Foundation
 import WebKit
 import RxSwift
+import RxCocoa
 
 protocol FxAViewProtocol: class, ErrorView {
     func loadRequest(_ urlRequest:URLRequest)
 }
 
-enum FxAError : Error {
-    case RedirectNoState, RedirectNoCode, RedirectBadState, EmptyOAuthData, EmptyProfileInfoData, UnexpectedDataFormat, Unknown
-}
-
 class FxAPresenter {
     private weak var view:FxAViewProtocol!
-    private let fxAActionHandler:FxAActionHandler
-    private let routeActionHandler:MainRouteActionHandler
-    private let store:FxAStore
+    fileprivate let fxAActionHandler:FxAActionHandler
```

Comment:
Nit: whitespace after the colon.

Having said that, you've used it fairly consistently.